selfie.c

```diff
@@ -9572,6 +10752,14 @@ uint64_t selfie() {
         return selfie_run(MONSTER);
       else if (string_compare(option, "-y"))
         return selfie_run(HYPSTER);
+      else if (string_compare(option, "-t")) {
```

Comment:
Let's use `-x86` as option.

Reply:
Please prepare an (optional, not among all) Makefile target `x86` that tests self-compilation using the x86 binary you generate. Then, add that target to the test targets of the docker image in travis.